app/assets/stylesheets/views/_service-manual-topic.scss

```diff
@@ -0,0 +1,9 @@
+@import "govuk_publishing_components/govuk_frontend_support";
+
+.related-item {
```

Comment:
I've noticed `_email_signup.html.erb` also uses `.related-item` but I didn't import the styles when I added it. Do we need to modify any styles for `_email_signup.html.erb` to match what you've done here, or can we just delete`.related-item` from that partial?

I added `_email_signup.html.erb` initially just for this one route https://www.gov.uk/service-manual/service-standard/

Reply:
Good shout! My new class was inadvertently adding a border part way through this column as well, when it shouldn't have. I've removed the classes and some of the wrapping divs in email_signup, as no longer needed 👍